src/main/scala/cromwell/engine/package.scala

```diff
@@ -14,15 +14,28 @@ import cromwell.binding.values.WdlValue
  *
  * Internally, this package is built on top of [[cromwell.binding]].
  */
-
 package object engine {
-  type WorkflowId = UUID
+  case class WorkflowId(id: UUID) {
+    override def toString = id.toString
+    def shortString = id.toString.split("-")(0)
+  }
```

Comment:
This wasn't ignored - it's just the newline I added is outside this commented block